java/deps.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
can we use `remote_file()` in the `BUCK` file and then leave it to buck to fetch these?